openfisca_core/tests/assets/parameters.yaml

```diff
@@ -0,0 +1,124 @@
+
+# Regular parameter values : meta + VALUES handling variation of its value in time
+- variable: smic_horaire_brut
+  description: SMIC horaire brut
+  format: float
+  type: monetary
+  VALUES:
+    - {deb: 2016-01-01, fuzzy: true,     valeur: 9.67}
+    - {deb: 2015-01-01, fin: 2015-12-31, valeur: 9.61}
+    - {deb: 2014-01-01, fin: 2014-12-31, valeur: 9.53}
+    - {deb: 2013-01-01, fin: 2013-12-31, valeur: 9.43}
+    # etc.
+
+#  Short date notation :
+#  Use it in the very common case where all VALUE leaves are consecutive
+- variable: plafond_securite_sociale
+  description: Plafond mensuel de la sécurité sociale
+  type: monetary
+  # Values should be written in descending ordered :
+  # what's most important today first
+  VALUES:
+    #  Fuzzy is explicit :
+    #  to express the disparition of a variable,
+    #  you should explicitely set its last date
+    #  (which is the first in the syntax) to 0
+    2017-01-01: 3218
+    # This first line above is computationally useless.
+    # But to humans with a limited capacity for watching parameter changes,
+    # it ensures that somebody checked the law and the value had not changed in 2017.
+    2016-01-01: 3218
+    2015-01-01: 3170
+    # Asking for a parameter for the date before the last will
+    # raise a ParameterNotFound error
+    # TODO: Or should it return 0 ?
+
+
+
+# BAREME node with 1 bracket...
+- variable: famille
+  description: Allocations familiales
+  BAREME:
```

Comment:
Yes that's one limitation of YAML parameters at the moment : only MarginalRateTaxScales are implemented